smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -182,6 +182,9 @@ class Program
         [Option("--parent-edge-device", Description = "Optional input to specify parent edge device id for nested edge scenario")]
         public string ParentEdgeDevice { get; } = string.Empty;
 
+        [Option("--overwrite-packages", Description = "Overwrite existing aziot packages with those specified in the bootstrapper")]
```

Comment:
Nit: Missing usage information in ExtendedHelpText above

Reply:
Fixed